emscripten/WebARKitJS.cpp

```diff
@@ -24,7 +24,7 @@ struct webARKitController {
   int height;
   int image2DSize;
   unsigned char *image2DFrame;
-  WebARKitOrbTracker *tracker;
```

Comment:
this was the root of all issues! not initializing the WebARKitOrbTracker as a pointer solved the isssue.